### PR TITLE
Create json rpc types by hand

### PR DESCRIPTION
**Pull request by tomclose**
Fixes https://github.com/statechannels/monorepo/issues/957

### CODE REVIEW

File: packages/channel-client/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
+{
```
Comment (geoknee) [inline packages/channel-client/.vscode/launch.json]: Does this belong in this PR?

File: packages/channel-client/.vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (geoknee) [inline packages/channel-client/.vscode/settings.json]: Does this belong?

File: packages/channel-provider/src/types.ts [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
 import {
-  CreateChannelResult,
-  CloseChannelResult,
-  UpdateChannelResult,
-  PushMessageResult,
-  JoinChannelResult,
-  GetAddressResult,
-  GetEthereumSelectedAddressResult,
-  ChallengeChannelResult,
-  BudgetResult1 as BudgetResult
```
Comment (geoknee) [inline packages/channel-provider/src/types.ts]: :+1:

File: packages/client-api-schema/package.json [hunk 4]
```diff
@@ -31,15 +17,32 @@
...
+  "devDependencies": {
+    "@types/eslint": "^6.1.3",
+    "@types/eslint-plugin-prettier": "^2.2.0",
+    "@typescript-eslint/eslint-plugin": "^2.10.0",
+    "@typescript-eslint/parser": "^2.10.0",
+    "eslint": "^6.7.2",
+    "eslint-plugin-prettier": "^3.1.1",
+    "json-schema-to-typescript": "^8.0.0",
+    "lint-staged": "9.4.0"
+  }
```
Comment (geoknee) [inline packages/client-api-schema/package.json]: Was it necessary to change these dependencies? Could this be the source of your linting woes?

Comment (tomclose) [reply packages/client-api-schema/package.json]: Hmm. This happened during a rebase. Will try and fix.

File: packages/channel-client/.vscode/settings.json (hunk 2 above)
Comment (tomclose) [reply packages/channel-client/.vscode/settings.json]: Probably not strictly, but I think it's probably ok (?) - it's just importing the `.vscode` functionality that probably should have set up when the package was created.

**Pull request merged**